client/src/components/App.js

```diff
@@ -26,6 +27,7 @@ const App = (props) => {
         <Route exact path="/">
           <h2>Hello from react</h2>
         </Route>
+        <Route exact path="/shows" component={ShowsList}></Route>
```

Comment:
convention when we are doing a "simple" route will be to have a self closing `Route` component, rather than an open and close Route component